app/views/admin/groups/_nav.html.erb

```diff
@@ -2,10 +2,10 @@
 <div class="panel-tab-wrapper">
   <ul class="nav nav-tabs">
     <% navigation_presenter.tabs.each do |tab| %>
-      <%= content_tag :li, role: 'presentation', class: tab.css_class do %>
-        <%= link_to tab.name, tab.path %>
-      <% end %>
+      <li class="nav-item" role="presentation">
```

Comment:
Reading Note: This appears to be moving the 'active' class to a link.  Which is likely alright.